src/lib/Regex.tsx

```diff
@@ -0,0 +1,2 @@
+export const passwordRegex =
+  /^(?=.*[A-Za-z])(?=.*\d)(?=.*[!@#$%^&*()_\-+={[}\]|:;"'<,>.?/`~])[\w!@#$%^&*()_\-+={[}\]|:;"'<,>.?/`~]{8,72}$/;
```

Comment:
```suggestion
  /^(?=.*[A-Za-z])(?=.*\d)(?=.*\W)[\S]{8,72}$/;
```
리팩토링이고 테스트 해봤습니다..
- \W: 특수문자
- \S: 공백 제외 모든 문자